app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -63,36 +65,6 @@ def supports_port?
     true
   end
 
-  def supported_api_versions
-    supported_api_versions_from_cache
-  end
-
-  def supported_api_versions_from_cache
-    cacher = Cacher.new(cache_key)
-    current_cache_val = cacher.read
-    force = current_cache_val.blank?
-    cacher.fetch_fresh(last_refresh_date, :force => force) { supported_api_versions_from_sdk }
-  end
-
-  def cache_key
-    "REDHAT_EMS_CACHE_KEY_#{id}"
-  end
-
-  def supported_api_versions_from_sdk
-    username = authentication_userid(:basic)
-    password = authentication_password(:basic)
-    probe_args = { :host => hostname, :port => port, :username => username, :password => password, :insecure => true }
-    probe_results = OvirtSDK4::Probe.probe(probe_args)
-    probe_results.map(&:version) if probe_results
-  rescue => error
-    # Note that errors when trying to find the supported API versions are perfectly normal, in particular authorization
-    # errors are expected, as in many situations, for example during discovery, the user name and the password aren't
-    # yet known. In these situations we *must* return an empty array, to indicate to the caller that it wasn't possible
-    # to determine the supported API versions.
-    _log.info("Can't determine the API versions supported by the server: #{error}")
-    []
-  end
-
   def supports_the_api_version?(version)
```

Comment:
why not to move other methods which deal with the supported api versions  ?

Reply:
done